tools/trap/HowToUseTrap.md

```diff
@@ -49,7 +49,10 @@ Device Drivers -> Block-to-character(BCH) Support
 ```
 
 ## How to parse RAMDUMP
-TRAP Script provides two interfaces: CUI and GUI
+TRAP Script provides two interfaces:  
+1. [CUI](#trap-using-cui)  
+2. [GUI](#trap-using-gui)  
+To get Debug Symbols/Crash point using assert logs [CUI](#to-display-debug-symbols/crash-point-using-assert-logs), enabling memory dumps [Build Steps](#how-to-enable-memory-dumps) is not mandatory.
```

Comment:
1. CUI Link is not working.
2. Let's add one empty line because currently it has indentation.